boards/v2/qemu/qemu_xtensa/board.cmake

```
@@ -2,7 +2,7 @@
 
 set(SUPPORTED_EMU_PLATFORMS qemu)
 
-if(CONFIG_BOARD_QEMU_XTENSA OR CONFIG_BOARD_QEMU_XTENSA_MMU)
+if(CONFIG_BOARD_QEMU_XTENSA OR CONFIG_BOARD_QEMU_XTENSA_DC233C_MMU)
```

`CONFIG_BOARD_QEMU_XTENSA_DC233C_MMU` is redundant when first check is board without identifier.
```suggestion
if(CONFIG_BOARD_QEMU_XTENSA)
```

---

Fixed